sql/addons_aggregates_v3.sql

```diff
@@ -0,0 +1,63 @@
+WITH
+  extended_addons AS (
+  SELECT
+  submission_date_s3 as submission_date,
+    sample_id,
+    client_id,
+    normalized_channel,
+    SUBSTR(app_version, 1, 2) AS app_version,
+    locale,
+    _addon.element.addon_id AS addon_id,
+    IF(_addon.element.foreign_install,
+      1,
+      0) AS is_foreign_install,
+    IF(_addon.element.is_system,
+      1,
+      0) AS is_system,
+    IF(_addon.element.is_web_extension,
+      1,
+      0) AS is_web_extension,
+    _addon.element.install_day AS install_day,
+    IF(_addon.element.addon_id IS NOT NULL
+      AND NOT _addon.element.is_system
+      AND NOT _addon.element.foreign_install
+      AND NOT _addon.element.addon_id LIKE '%mozilla%'
+      AND NOT _addon.element.addon_id LIKE '%cliqz%'
+      AND NOT _addon.element.addon_id LIKE '%@unified-urlbar%',
+      1,
+      0) AS is_self_install,
+    IF(_addon.element.addon_id LIKE "%@shield.mozilla%",
+      1,
+      0) AS is_shield_addon,
+    profile_creation_date
+  FROM
+    telemetry.main_summary_v4,
+    UNNEST(active_addons.list) AS _addon
+  WHERE
+    submission_date_s3 = @submission_date
+    AND client_id IS NOT NULL)
+SELECT
+  DISTINCT submission_date,
```

Comment:
for this select distinct to do the same thing as https://github.com/mozilla/python_mozetl/blob/master/mozetl/addon_aggregates/addon_aggregates.py#L141 it needs to move to line 4.